File: ForumFrontend/forumfrontend/src/interfaces/Comment.ts
export interface Comment {
	id: number;
	content: string;
	userId: string;
	username: string;
	dateOfCreation: Date;
	postId: number;
}

export type CreateCommentData = {
	content: string;
	//userId: string;
	postId: Number;
	//dateOfCreation: Date
};

export type UpdateCommentData = {
	id: number;
	content: string;
};

File: ForumBackend/ForumBackend/Models/Comment.cs
using System.ComponentModel.DataAnnotations;

namespace ForumBackend.Models
{
    public class Comment
    {
        public int Id { get; set; }
        [Required]
        public string Content { get; set; } = string.Empty;
        public DateTime DateOfCreation { get; set; } = DateTime.Now;
        public int PostId { get; set; }
        public Post Post { get; set; } = null!;

        public string UserId { get; set; } = string.Empty;
        public User User { get; set; } = null!;
    }
}
